### DISCUSSION TITLE
Questions about participating teams in the AAAI2023 Knowledge Tracing  Competition

### DISCUSSION THREAD

**Discussion by XingYing-stack** [pykt-team/pykt-toolkit#86] [Q&A] [answered]
Hello, I noticed that there seems to be no real-name authentication for registration in the AAAI2023 knowledge tracking competition. How to ensure that a user will not use multiple accounts? What if the first player opens multiple accounts so that he/she can take over the top three teams?

**Comment by gaozhanfire** [comment]
I have the same question.

**Comment by meta-tabchen** [comment, marked as answer]
Thank for your attention to our challenge,  we will ask participants to submit their team details, and for the top teams, we will review their codes to make the final result reliable. The details we will send one email to each participant.

**Reply by may248110** [reply]
Does the submission limit apply to whole team or to individual team members?

**Reply by meta-tabchen** [reply]
For the public leaderboard, the limit applies to individual members, because it is hard to limit users in teams to create new accounts. But for the final result(private leaderboard), the limit (3 times) is to the whole team, for each team we will only choose one account(the account decided by the team) to calculate the best score as the final score.

**Reply by may248110** [reply]
Thank you very much for the clarification!
